doc/rst/usingchapel/editors/emacs.rst

```diff
@@ -0,0 +1,69 @@
+.. _emacs:
+
+Emacs
+-----
+
+.. _emacs-syntax:
+
+Syntax Highlighting
+~~~~~~~~~~~~~~~~~~~
+
+To get syntax highlighting for Chapel in Emacs, you can use the files in the top-level ``highlight/emacs`` directory of the Chapel source tree.  These files are licensed under the GPL, see ``highlight/emacs/LICENSE``.  To use them, take the following steps:
```

Comment:
This seems to be duplicating some but not all of the `highlight/emacs/README.rst` file.  I definitely understand not wanting to touch the (long out of date) information about submitted versions, but I worry a little about code drift.  I'd be tempted to do some linking magic, but don't feel the need to spend too much effort on it

Reply:
I was kinda temped to just delete most of the contents of `highlight/emacs/README.rst` and just link to the new docs, rather than try and keep them in sync

Reply:
I think there's some information that it would be good to maintain there:
- it potentially still having hiccups
- it being contributed by Steve Balensiefer (which is a name I don't recognize at all and might have been an outside contributor?  I'd ask Brad on that)
- the reference to the COPYING file which still exists

Reply:
But I'm otherwise sympathetic to deleting the duplicated information